{
    "Computer":  "VSDC1",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1759605779753)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Error calling a routine on a Shadow Copy Provider {74600e39-7dc5-4567-a03b-f091d6c7b092}. Routine details BeginPrepareSnapshot({93719f4e-5846-42b6-8d08-bc1ef878a5ae},{1157a33b-2cee-464c-ac1a-8c1c575cfc00},\\\\?\\Volume{c6def331-2795-4a00-bf35-64cc5b01fbec}\\) [hr = 0x8000ffff, Catastrophic failure\r",
                       "Id":  12293,
                       "ErrorCode":  "0x8000ffff",
                       "UpdateTitle":  null,
                       "EventRecordID":  80289
                   },
                   {
                       "TimeCreated":  "\/Date(1759604742434)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Unexpected error CreateFileW(\\\\?\\Volume{f3294ed6-a154-11f0-ab87-00155d611102},0x80000000,0x00000003,...).  hr = 0x80070037, The specified network resource or device is no longer available.\r",
                       "Id":  12289,
                       "ErrorCode":  "0x80000000",
                       "UpdateTitle":  null,
                       "EventRecordID":  80281
                   },
                   {
                       "TimeCreated":  "\/Date(1759604742137)\/",
                       "Source":  "Volsnap",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The shadow copies of volume \\\\?\\Volume{f3294ed8-a154-11f0-ab87-00155d611102} were aborted during detection because a critical control file could not be opened.",
                       "Id":  27,
                       "ErrorCode":  "0x0001",
                       "UpdateTitle":  null,
                       "EventRecordID":  194070
                   },
                   {
                       "TimeCreated":  "\/Date(1759566231583)\/",
                       "Source":  "Volsnap",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The shadow copies of volume \\\\?\\Volume{64d0d9ac-a0fb-11f0-ab87-00155d611102} were aborted because the shadow copy storage volume was not present.",
                       "Id":  32,
                       "ErrorCode":  "0x0005",
                       "UpdateTitle":  null,
                       "EventRecordID":  193302
                   },
                   {
                       "TimeCreated":  "\/Date(1759559680128)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Windows cannot load the extensible counter DLL \"C:\\Windows\\System32\\iasperf.dll\" (Win32 error code The specified module could not be found.).",
                       "Id":  1023,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  79687
                   },
                   {
                       "TimeCreated":  "\/Date(1759542221438)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{a860543f-91cd-4694-9557-d6266e9a1317}\" of \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\"  task for user \"NT AUTHORITY\\SYSTEM\" . Additional Data: Error Value: 2147946720.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152360
                   },
                   {
                       "TimeCreated":  "\/Date(1759542221438)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\" , instance \"{a860543f-91cd-4694-9557-d6266e9a1317}\" , action \"\" . Additional Data: Error Value: 2147946720.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152359
                   },
                   {
                       "TimeCreated":  "\/Date(1759539830515)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\Workplace Join\\Automatic-Device-Join\" task for user \"(default user)\". Additional Data: Error Value: 2147943568.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152175
                   },
                   {
                       "TimeCreated":  "\/Date(1759539786277)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Access to performance data was denied to user \"SYSTEM\" (value from GetUserName() for the running thread) as attempted from module \"C:\\Program Files (x86)\\NinjaOne\\NinjaRMMAgent.exe\" (value from GetModuleFileName() for the binary that issued the query).",
                       "Id":  1000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  79384
                   },
                   {
                       "TimeCreated":  "\/Date(1759539771058)\/",
                       "Source":  "Microsoft-Windows-DistributedCOM",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "DCOM got error \"1115\" attempting to start the service TrustedInstaller with arguments \"Unavailable\" in order to run the server:\r",
                       "Id":  10005,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  192493
                   }
               ],
    "LastPoll":  "10/04 15:47"
}